src/otg/assets/schemas/study_locus.json

```diff
@@ -82,6 +88,18 @@
       "nullable": true,
       "type": "string"
     },
+    {
+      "metadata": {},
+      "name": "credibleSetIndex",
```

Comment:
What is the interpretation of this index? I feel that we've talked about this already, can you remind me if credible set 1 means anything over credible set 2?